distutils/sysconfig.py

```diff
@@ -340,6 +340,7 @@ def customize_compiler(compiler):
 
         ldshared = _add_flags(ldshared, 'LD')
         ldcxxshared = _add_flags(ldcxxshared, 'LD')
+        cflags = os.environ.get('CFLAGS', cflags)
         cflags = _add_flags(cflags, 'C')
```

Comment:
Hi, I am a bit confused why you are keeping the `_add_flags(cflags, 'C')` line, CXX doesn't have it (should it?), and it seems redundant (probably duplicates those flags, although I didn't test)?

Reply:
No, you're right, I'd left it in because I tried something else, and forgot to clean it up. I'll do that in a followup after testing it.

Reply:
Thanks, submitted https://github.com/pypa/distutils/pull/325. Sorry for the oversight.